OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -152,7 +152,7 @@ public interface INotifyDelivery { void IncomingDelivery(Actor self); void Deliv
 	public interface INotifyDocking { void Docked(Actor self, Actor harvester); void Undocked(Actor self, Actor harvester); }
 
 	[RequireExplicitImplementation]
-	public interface INotifyResourceAccepted { void OnResourceAccepted(Actor self, Actor refinery, int amount); }
+	public interface INotifyResourceAccepted { void OnResourceAccepted(Actor self, Actor refinery, string resourceType, int amount, int value); }
```

Comment:
It would be nice to have documentation explaining the difference between `amount` and `value`.

Reply:
Agreed that `amount` is ambiguous, and outside our usual naming scheme. I renamed it to `count` to be clearer.